# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/UpdateImageHelper.h
# Commit: c09a51622e98
# Full Hash: c09a51622e988e81077c75e3aaa5a49df66284b3
# Author: Doug Thayer <dothayer@mozilla.com>
# Date: 2019-03-22 16:44:08
# Regressor Bug: 1441308
# File Overlap Count: 2
# Description:
#   Bug 1441308 - Core renderroot splitting changes r=kats,sotaro
#   
#   This is a large patch that contains all of the core changes for
#   renderroot splitting.
#   
# ==============================================================================

diff -r 737807563dd5 -r c09a51622e98 gfx/layers/UpdateImageHelper.h
--- a/gfx/layers/UpdateImageHelper.h	Thu Mar 21 23:14:26 2019 +0000
+++ b/gfx/layers/UpdateImageHelper.h	Fri Mar 22 03:15:14 2019 +0000
@@ -52,7 +52,7 @@
     return target.forget();
   }
 
-  bool UpdateImage() {
+  bool UpdateImage(wr::RenderRoot aRenderRoot) {
     if (!mTexture) {
       return false;
     }
@@ -65,7 +65,8 @@
     RefPtr<TextureWrapperImage> image = new TextureWrapperImage(
         mTexture, gfx::IntRect(gfx::IntPoint(0, 0), mImageSize));
     mImageContainer->SetCurrentImageInTransaction(image);
-    return mImageClient->UpdateImage(mImageContainer, /* unused */ 0);
+    return mImageClient->UpdateImage(mImageContainer, /* unused */ 0,
+                                     Some(aRenderRoot));
   }
 
  private: